### ISSUE TITLE
Request: Port as Firefox webextension.

### ISSUE THREAD

**Issue by Hetflik** [Cortys/comic-backup#74]

**Comment by gregoryopera** [comment]
Yes!

Currently I run two browsers - Chromium and Mozilla Firefox, though the former is primarily only used for "ripping" comics... It'd be great if I could stick to just Firefox.

I'm not a developer, but my understanding is that one of the advantages for the WebExtensions API is that extensions can be more easily and more quickly ported from other browsers, such as Chromium/Google Chrome.

Is this not the case?

**Comment by Hetflik** [comment]
Yes. WebExtensions were designed for cross-browser compatibility... For some basic WE the only thing you need to do is change the namespace from 'chrome' to 'browser' (That's not even necessary, Firefox does it on it's own). This is unforunately not the case with this one. 
There will be problem with options_page in manifest.json, which is not supported in Firefox...
Paths to script files might also need some changes. But still porting shouldn't be huge problem for the developer.

**Comment by BucketListMod** [comment]
I would like to second this, as well! I'm not a coder really, but I do fiddle, and if I could be pointed towards resources for how to do the conversion myself, I'd be happy to do so and then add the code as a different branch!
